### ISSUE TITLE
Request: Adding conversion of +- to ±

### ISSUE THREAD

**Issue by wagerc97** [mgmeyers/obsidian-smart-typography#61]

**Comment by c-lipka** [comment]
I second this request in general, but would suggest "+/-" as the sequence to be replaced by "±"

**Comment by AnonJohn** [comment]
+1 to c-lipka. +/- is more natural to me and is used in at least one other program (zim-wiki)

**Comment by wit1605** [comment]
I don't think this is getting any updates since it's been 3 years, but it's quite easy to add it yourself.

in .obsidian/plugins/obsidian-smart-typography open main.js in a text editor.
Pick a 'const' of which you've turned the rules on, probably comparisonRules.
add this after the comma of the last thing in the list:
for +/-

```
    {
        trigger: "-",
        from: "+/-",
        to: "±",
        contextMatch: /(?:^|\s)\+\/$/,
    },

```
or this for +-
```
    {
        trigger: "-",
        from: "+-",
        to: "±",
        contextMatch: /(?:^|\s)\+$/,
    },
```

turn the plugin off and back on again. If it fails to turn back on, check if your commas match up properly.
